ui/src/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Why is this file needed? I think there are already two tsconfig.json files under `ui/`, though one might be tsconfig.app.json or something.